tools/src/test/java/org/apache/kafka/tools/TransactionsCommandTest.java

```diff
@@ -187,14 +187,25 @@ private void testDescribeProducers(
         assertEquals(expectedRows, new HashSet<>(table.subList(1, table.size())));
     }
 
-    @Test
-    public void testListTransactions() throws Exception {
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void testListTransactions(boolean hasDurationFilter) throws Exception {
         String[] args = new String[] {
             "--bootstrap-server",
             "localhost:9092",
             "list"
         };
 
+        if (hasDurationFilter) {
+            args = new String[] {
+                "--bootstrap-server",
+                "localhost:9092",
+                "list",
+                "--duration-filter",
+                Long.toString(Long.MAX_VALUE)
```

Comment:
hmm this will not return anything, right?

Reply:
This just tests the cli cmd parsing parameters correctly. It will not call actual list transaction.